### ISSUE TITLE
Import SVG to existing font adds lots of extra nodes

### ISSUE THREAD

**Issue by joebland** [fontforge/fontforge#2979]
Issue:
[x ] Screenshot 
https://nimbus.everhelper.me/client/notes/share/695547/aeip2ssm48hr5zwmd7i8

[x] The FontForge version and the operating system you're using
AWST 20:55 4-Oct-2016 Windows 10

[x] The behavior you expect to see 
SVGs import from Illustrator CS6 into roboto-light.ttf, which I have opened in FF, and maintain the same nodes as they have in Illustrator.

[x] The actual behavior
Many new nodes are added to shapes imported as SVG or EPS.

[x] Steps to reproduce the behavior
- Download the two sample files at https://nimbus.everhelper.me/client/notes/share/695547/aeip2ssm48hr5zwmd7i8
- Open Roboto-Light.ttf in FontForge, go to the 'o' glyph window
- Import 'o.svg', note that in Illustrator there are only 8 nodes in the shape, but in FF there are many more.

[x] Possible solution/fix/workaround
Importing the svg into a completely new font file fixes the issue. But I want to create the new font to take advantage of many features completed already in Roboto (the license allows this), not start from scratch.
If the issue can't be solved, is there a way to replicate Roboto to create a new font, which will import the SVGs cleanly?

**Comment by joebland** [comment]
Think I solved this one. 

True Type fonts use a certain kind of bezier curves which are different to Post Script curves. Curves in SVG files are the Post Script kind, where as a TTF font file uses the other kind. Therefore when I place the SVG into the Roboto-Light.TTF file it adds nodes to the SVG curves in an attempt to match my curves in it's format.

The solution I found is to open the TTF file in Font Forge, then export an OTF font file of it. Then open that OTF file in Font Forge and you're good to go with your SVG curves as drawn in Illustrator/Designer/Inkscape. 

Might be better to use the TTF curve kind if you were starting a new font and not using another drawing program, but seems to work and gets the outcome I want (least number of nodes).
